modules/certificate/certificates.permission.js

```diff
@@ -0,0 +1,19 @@
+const { hasRoles } = require('../../utils/rules');
+const {
+  roles: { USER, ADMIN, SUPERADMIN },
+} = require('../../consts');
+
+const certificatePermissionsQuery = {
+  getCertificateById: hasRoles([USER, ADMIN, SUPERADMIN]),
```

Comment:
why do you need this permission if you don't need to restrict by roles? use other middleware

Reply:
add validation middleware

Reply:
created new issue [https://github.com/ita-social-projects/horondi_client_be/issues/1040](https://github.com/ita-social-projects/horondi_client_be/issues/1040)